Source/ACE.Entity/Position.cs

```diff
@@ -1,38 +1,38 @@
-﻿namespace ACE.Entity
-{
-    using System;
-    using System.IO;
-    using Enum;
-    using Common;
-    using MySql.Data.MySqlClient;
+﻿using System;
+using System.IO;
+using ACE.Entity.Enum;
+using ACE.Common;
+using MySql.Data.MySqlClient;
 
+namespace ACE.Entity
+{
     [DbTable("ace_position")]
     [DbGetList("ace_position", 23, "aceObjectId")]
-    public class Position
+    public sealed class Position
```

Comment:
mainly because refactor was barking at me as a suggestion.   I forget the exact wording but something like virtual method called in constructor?   I can put it back - it seems like a reasonable suggestion to me so I did it.